receiver.go

```diff
@@ -45,14 +47,30 @@ func IsForcePush(hook *HookInfo) (bool, error) {
 	return base != hook.OldRev, nil
 }
 
+func (r *Receiver) CheckAllowedBranch(hook *HookInfo) error {
+	if r.MasterOnly { // for BC
+		r.AllowedBranches = append(r.AllowedBranches, "refs/heads/master")
```

Comment:
Maybe we should use `AllowedRefs` instead since we're including `refs/<type>/<name>` in the list. Could also store these in a map so there's no need to use an extra package.

Reply:
I didn't get to this earlier, but I'll have a look in the next 2 weeks.